.github/workflows/build-freebsd.yml

```diff
@@ -0,0 +1,36 @@
+name: BE13_API CI (c++17) on FreeBSD
+on: [push]
+
+jobs:
+  build:
+    runs-on: macos-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - name: c/c++ test
+      uses: vmactions/freebsd-vm@v0.1.4
+      with:
+        usesh: true
```

Comment:
What does `usesh` do? I can't find documentation for it. 
I did find this, though:
* https://github.com/marketplace/actions/freebsd-vm

Reply:
> What does `usesh` do? I can't find documentation for it.

Not sure, but probably it makes script running with `/bin/sh` instead of `bash`? Sorry, I'm too lazy to check right now, I've just copied this line from the original example. :-)

> I did find this, though:
> 
> * https://github.com/marketplace/actions/freebsd-vm

Yes, it is the same thing.